src/s3ql/http.py

```diff
@@ -0,0 +1,1558 @@
+'''
+http.py - this file is part of S3QL.
+
+Copyright © 2008 Nikolaus Rath <Nikolaus@rath.org>
+
+This work can be distributed under the terms of the GNU GPLv3.
+
+The CaseInsensitiveDict implementation is copyright 2013 Kenneth Reitz and
+licensed under the Apache License, Version 2.0
+(http://www.apache.org/licenses/LICENSE-2.0)
```

Comment:
Thanks for checking! I think that only applies to projects that are entirely licensed under this license. For individual files, https://www.apache.org/licenses/LICENSE-2.0 says at the end "To apply the Apache License to specific files in your work, attach the following boilerplate declaration [...]".

I'd rather not include the full license to avoid confusion about which license applies overall.